data/items/items.xml

```diff
@@ -37547,9 +37547,18 @@
 	<item id="27046" article="an" name="ancient coin" />
 	<item id="27047" article="a" name="coral brooch" />
 	<item id="27048" article="a" name="gemmed figurine" />
-	<item id="27049" article="a" name="pillow backpack" />
+	<item id="27049" article="a" name="pillow backpack">
+		<attribute key="weight" value="1450" />
+		<attribute key="containerSize" value="24" />
+		<attribute key="slotType" value="backpack" />
+	</item>
 	<item id="27050" article="an" name="orclops santa" />
-	<item id="27051" article="a" name="birthday backpack" />
+	<item id="27051" article="a" name="birthday backpack">
+		<attribute key="weight" value="1500" />
+		<attribute key="containerSize" value="20" />
+		<attribute key="slotType" value="backpack" />
+		<attribute key="description" value="It was awarded during the festivities of Tibia's 20th anniversary." />
```

Comment:
```suggestion
		<attribute key="description" value="It was awarded during the festivities of 20th anniversary." />
```

Reply:
"[...] the festivities of **the** 20th anniversary.", right?

Reply:
TFS uses worlds instead of Tibia in the descriptions, you can see this in other items, so I decided to follow the same pattern.